smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/EnumGenerator.java

```diff
@@ -35,10 +35,11 @@
  * <p>We will generate the following:
  *
  * <pre>{@code
- * export enum TypedYesNo {
+ * export const /* enum *\/ TypedYesNo {
```

Comment:
Should this be the following?

```suggestion
 * export const /* enum *\/ TypedYesNo  = {
```

Reply:
I added the missing `=`, but the extra slash is needed, otherwise `*/` is interpreted as the close of the multi-line comment.